app-text/poppler/poppler23-23.09.0.recipe

```diff
@@ -253,16 +253,18 @@ REQUIRES="
 	lib:libcurl$secondaryArchSuffix
 	lib:libfontconfig$secondaryArchSuffix
 	lib:libfreetype$secondaryArchSuffix
+	lib:libgpgmepp$secondaryArchSuffix
 	lib:libiconv$secondaryArchSuffix
 	lib:libintl$secondaryArchSuffix
 	lib:libjpeg$secondaryArchSuffix
 	lib:liblcms2$secondaryArchSuffix
-#	lib:libnspr4$secondaryArchSuffix
-#	lib:libnss3$secondaryArchSuffix
-#	lib:libnssutil3$secondaryArchSuffix
+	lib:libnspr4$secondaryArchSuffix
+	lib:libnss3$secondaryArchSuffix
+	lib:libnssutil3$secondaryArchSuffix
```

Comment:
Not sure what changed since earlier checks, but it now doesn't seem to have a problem with libnss3. :)

Err ... nope, still an issue (will disable later)
```
~> TeXstudio 
runtime_loader: /boot/system/lib/libpoppler.so.131.0.0: Version dependency "/packages/nss-3.73.1-1/.self/develop/lib/libsmime3.so" not found
```